### DISCUSSION TITLE
New 3-hourly maps coming soon

### DISCUSSION THREAD

**Discussion by matteason** [matteason/daily-cloud-maps#9] [Announcements]
Hi all,

Apologies that this project has been so unreliable lately. There have been a lot of problems with the source data missing parts, or sometimes not being available at all.

I've been trying to work out a way to use an alternative data source (EUMETSAT) for a while. The main challenge has been that the whole-Earth data they provide is infrared-based, so processing this into something that matches the visible light-based data this project currently uses has been a challenge.

Luckily I've finally found a method that results in quality that I'm happy with, so in the next week or so I hope to lauch a new 'live cloud maps' service. The new data source is updated every 3 hours (rather than every 24 hours at the moment) so I'm able to provide a nearly-live service. Another benefit is that because the data is from geostationary satellites, the whole Earth is captured simultaneously, meaning the images don't contain the same vertical banding that you may have seen with the current data.

This new service will live in a new GitHub project, but the images will still be available from the URLs you may already be using from this project. I'll post again when this change is live.

I'm hopeful that this will result in a much more reliable service going forwards. Here's a sample of the new imagery:

![Clouds over a flat image of the Earth](https://github.com/matteason/daily-cloud-maps/assets/1935173/c2a22c75-d651-4a6c-9fc9-8d253f35ce6d)

It's still lacking a bit of detail compared to the old data, but I'll continue to look for ways to improve it.

For comparison, this is the old imagery from the same date:
![Clouds over a flat image of the Earth](https://github.com/matteason/daily-cloud-maps/assets/1935173/657615a6-cbd2-4383-b5d0-ce33c67295c9)

**Comment by wlbr** [comment]
Hi,

while I can retrieve the current cloud files using 
[`https://github.com/matteason/daily-cloud-maps/releases/download/2022-07-27/8192x4096-clouds.jpg`](https://github.com/matteason/daily-cloud-maps/releases/download/2022-07-27/8192x4096-clouds.jpg) , 
this does not seem to work with the "latest" link at
[`https://matteason.github.io/daily-cloud-maps/8192x4096-clouds.jpg`](https://matteason.github.io/daily-cloud-maps/8192x4096-clouds.jpg) 
There is only the dummy cloud part on antarctica.

Am I doing something wrong?

Regards,
Michael

**Comment by matteason** [comment]
Hi Michael,

The new 3-hourly service is up and running at https://github.com/matteason/live-cloud-maps and should be much more reliable. The new URL for that image is https://clouds.matteason.co.uk/images/8192x4096/clouds.jpg - let me know if you have any issues with it!
